src/Protocol/ProtocolRecognizer.h

```diff
@@ -20,6 +20,7 @@
 // Adjust these if a new protocol is added or an old one is removed:
 #define MCS_CLIENT_VERSIONS "1.8.x, 1.9.x, 1.10.x, 1.11.x"
 #define MCS_PROTOCOL_VERSIONS "47, 107, 108, 109, 110, 210, 315, 316"
+#define MCS_LATEST_PROTOCOL_VERSION "316"
```

Comment:
I'm not 100% sure with how the JSON writer works, but won't this being a string cause the protocol version to be sent as a string, instead of a number?

Shouldn't this be just

```c++
#define MCS_LATEST_PROTOCOL_VERSION 316
```
?

Minecraft's own JSON parser will silently correct it, but I'm pretty sure that that other 3rd-party programs won't be as happy about it.